Remotion/ObjectBinding/Web.ClientScript/Scripts/BocAutoCompleteReferenceValue.jquery.ts

```diff
@@ -1512,12 +1577,12 @@
       }
 
       var popUpDiv = popUp.children ('div');
-      var contentHeight = Math.max(0, Math.max(popUpDiv.children().map(function () { return this.offsetHeight + this.offsetTop; }).get()));
+      var contentHeight = Math.max(0, Math.max(parseInt(String(popUpDiv.children().map(function (this: JQuery) { return this.offsetHeight + this.offsetTop; }).get()))));
 
       var contentWidth = popUp.data('popUpContentWidth');
       if (!isVisibe)
       {
-        contentWidth = Math.max(0, Math.max(popUpDiv.children().map(function () { return this.offsetWidth + this.offsetLeft; }).get()));
+        contentWidth = Math.max(0, Math.max(parseInt(String(popUpDiv.children().map(function (this: JQuery) { return this.offsetWidth + this.offsetLeft; }).get()))));
```

Comment:
These are both issues and we should for now do a `as any` cast. The conversion to string and parsing back to number has a different behavior from the previous code. But the previous code is already broken since an array is passed to `Math.max` (without a spread operator). 

This has the following wonderful behvior:
```js
Math.max( [] ); // 0
Math.max( [34] ); // 34
Math.max( [34, 12] ); // NaN
```

This should be a bug issue

Reply:
Thanks for the nice examples, I'll create a ticket right away